static/app/views/settings/projectMetrics/metricsExtractionRuleForm.tsx

```diff
@@ -331,13 +339,32 @@ export function MetricsExtractionRuleForm({
                   )}
                 />
               }
-              placeholder={t('Select span attribute')}
-              creatable
-              formatCreateLabel={value => `Custom: "${value}"`}
               onCreateOption={value => {
                 setCustomAttributes(curr => [...curr, value]);
                 model.setValue('spanAttribute', value);
               }}
+              components={{
+                MenuList: (
+                  menuListProps: React.ComponentProps<typeof components.MenuList>
+                ) => {
+                  return (
+                    <MenuList
+                      {...menuListProps}
+                      info={tct(
+                        'Select an attribute or create one. [link:See how to instrument a custom attribute.]',
+                        {
+                          link: (
+                            <ExternalLink href="https://docs.sentry.io/product/explore/metrics/metrics-set-up/" />
+                          ),
+                        }
+                      )}
+                    />
+                  );
+                },
+              }}
+              placeholder={t('Select span attribute')}
+              creatable
+              formatCreateLabel={value => tct('Create "[value]"', {value})}
```

Comment:
I checked this out and we probably don't need this prop anymore since the default is `Create [value]` (same for other occurences)

Reply:
the new version has double quotes, but I don't have a strong opinion on this.